modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java

```diff
@@ -1853,11 +1866,30 @@ private KBGroupServiceConfiguration _getKBGroupServiceConfiguration(
 			new GroupServiceSettingsLocator(groupId, KBConstants.SERVICE_NAME));
 	}
 
+	private ServiceContext _getMinimalServiceContext(
```

Comment:
Why not simply call this method `_getServiceContext()`? Adding the "minimal" qualifier to it implies that there's something special about it. If there were multiple acceptable ways of initializing a kb service context this would be fine, but as there's only this one I think it is a bit confusing.

Reply:
OK